src/quemb/kbe/pbe.py

```diff
@@ -665,7 +663,7 @@ def initialize(self, eri_, compute_hf, restart=False):
 
             # energy
             if compute_hf:
-                eh1, ecoul, ef = self.Fobjs[frg].energy_hf(return_e1=True)
+                self.Fobjs[frg].energy_hf(return_e1=True)
```

Comment:
it would be a great idea to refactor `energy_hf` to actually return nothing.
After all it is a method with side-effects on `self` and this would make it explicit.

(If this is not possible because the returned value is actually used somewhere else, then I would set at least `return_e1=False` here)